frontend/gateway/pb/gateway.proto

```diff
@@ -48,6 +48,7 @@ message Result {
 		RefMap refs = 4;
 	}
 	map<string, bytes> metadata = 10;
```

Comment:
Could attestations be serialised and passed under this existing field instead?

Reply:
Unfortunately not. The attestation struct needs to have references to parts of the build graph, similar to `Ref`/`RefMap` - and at each layer through the frontend/client/exporter, we need to transform them appropriately, so these aren't really possible to serialise.

Reply:
There are size limitations in here. These objects could potentially be in MB and can't be inlined in proto requests because of it.

Keeping refs also keeps the lazy semantics of BuildKit where you can pass objects between solve requests without actually forcing evaluation, resulting in efficient solve and cache semantics.